diff --git a/dist/lib/helpers/get-registry.js b/dist/lib/helpers/get-registry.js
index 1d793074fc7efda458c3eded4c4519572c151bb6..c8690aa3bf0a5a05800200979952a25306dc5878 100644
--- a/dist/lib/helpers/get-registry.js
+++ b/dist/lib/helpers/get-registry.js
@@ -21,7 +21,7 @@ function getRegistry(baseDir = process.cwd()) {
     const resolvedFlags = pkgManager === 'npm' ? '--no-workspaces' : '';
     let registry = `https://registry.npmjs.org/`;
     try {
-        const output = (0, _child_process.execSync)(`${pkgManager} config get registry ${resolvedFlags}`, {
+        const output = (0, _child_process.execSync)(`${pkgManager} config get npmRegistryServer ${resolvedFlags}`, {
             env: {
                 ...process.env,
                 NODE_OPTIONS: (0, _utils.getFormattedNodeOptionsWithoutInspect)()
